build-tasks/templates/rule.template.js

```diff
@@ -3,12 +3,6 @@ module.exports = ({ ruleName, walkerName }) =>
 import * as Lint from 'tslint';
 
 import {ExtendedMetadata} from './utils/ExtendedMetadata';
-// use (and contribute to) AstUtils for common AST functions // TODO: delete comment
-import {AstUtils} from './utils/AstUtils';
-// use Utils instead of Underscore functions // TODO: delete comment
-import {Utils} from './utils/Utils';
-
```

Comment:
Removed the imports as well as they were unused.

Reply:
I reverted the changes in this file in a manually made commit so if you're interested in sending them in later, they'll still be here. 👍